### ISSUE TITLE
Fix this source https://www.royalroad.com/

### ISSUE THREAD

**Issue by draillihtnek** [dipu-bd/lightnovel-crawler#1248]
Novel URL:
https://www.royalroad.com/fiction/35398/monroe

App Location: installed via pip3 on Linux Mint 19.1
App Version: Lightnovel Crawler 2.28.11

Describe this issue: 
Crawling fails for all royalroad.com novels I tried. Copied from my terminal window:
```➡ Press  Ctrl + C  to exit

? Enter novel page url or query novel: https://www.royalroad.com/fiction/35398/monroe
Retrieving novel info...
https://www.royalroad.com/fiction/35398/monroe

 ❗ Error: 'NoneType' object is not subscriptable 


--------------------------------------------------------------------------------
 🔗  https://github.com/dipu-bd/lightnovel-crawler/issues 
================================================================================
```
Been using this on royalroad.com for quite a while. Guessing they changed something?

**Comment by ebausson** [comment]
Fixed in #1249 , on dev branch.

**Comment by draillihtnek** [comment]
Working in v2.29.1, thanks for the quick fix!
